fundingmanager.go

```diff
@@ -630,7 +633,9 @@ func (f *fundingManager) Start() error {
 			go func(dbChan *channeldb.OpenChannel) {
 				defer f.wg.Done()
 
-				err := f.handleFundingConfirmation(dbChan, shortChanID)
+				err := f.handleFundingConfirmation(
+					nil, dbChan, shortChanID,
```

Comment:
not sure if passing `nil` here is the best option, can make it easy to make errors down the line. Instead, can you fetch the peer (`NotifyWhenOnline `) right before it is used, or make sure that it will never be passed in as `nil`?

Reply:
Changed it so that we fetch the peer before `handleFundingConfirmation`.